Homework_2.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Microsoft.VisualBasic;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Kristina_Kulich__Application;
+
+public class Cheque : IEquatable<Cheque>
+{
+    public string Productname { get; set; }
+    /*private decimal _price;
+    public string Price
+    {
+        get { return string.Format("{0:C}", _price); }
+    }*/
+    public double Quantity { get; set; }
+    public decimal Price { get; set; }
+    public decimal Sum { get; set; }
+
+
+    /*public double Sum
+    {
+        get { return sum; }
+        set { sum = this.CalculateSum(); }
+    }*/
+    public bool Equals(Cheque? other)
+    {
+        throw new NotImplementedException();
+    }
+
+    public decimal CalculateSum()
+    {
+        Sum = (decimal)Quantity* Price;
+        return Sum;
+    }
+
+    public override string ToString()
+    {
+        return  Productname + "    $" + Price + "   Quantity:" + Quantity  ; // + " Sum: $ can't make sum be calculated up to here
+    }
+}
+
+public class Homework_2
+{
+    public void Task_1()
+    {
+        Console.WriteLine("Input your favorite Book: "); //1-string
+        string book = Console.ReadLine();
+        Console.WriteLine(String.Format("{0} is your favorite book", book));
+        
+        Console.WriteLine("Input your average note:"); //2-double
+        double note = Convert.ToDouble(Console.ReadLine());
+        Console.WriteLine($"You've got {note} for 9th grade");
+        
+        Console.WriteLine("Input Belarus population: "); //3- long
+        long population = Convert.ToInt64(Console.ReadLine());
+        Console.WriteLine($"Belarus population is:{population:n0}"); 
+        
+        Console.WriteLine("Input your favorit symbol:"); //4-char
+        char symbol = Convert.ToChar(Console.ReadLine());
+        Console.WriteLine($"your favorite symbol category is: {symbol} {CharUnicodeInfo.GetUnicodeCategory(symbol)}");
+        string c = String.Concat(symbol, population);
+        Console.WriteLine(c);
+    }
+
+    public void Task_2()
+    {
+        Console.WriteLine("Input your heigh in meters:");
+        decimal.TryParse(Console.ReadLine(), out decimal height);
+        Console.WriteLine(String.Format("Your height is: {0}m", height));
+        
+        Console.WriteLine("Input your favorite number from 1-10:");
+        short.TryParse(Console.ReadLine(), out short luckynumber);
+        Console.WriteLine(String.Format("Your lucky number is: {0}", luckynumber));
+
+    }
+
+    public void Task_3()
+    {
```

Comment:
There should be an additional output for success/failure of operation :)

Reply:
fixed